app/assets/stylesheets/application.css.scss

```diff
@@ -34,15 +40,59 @@ body {
 
 .row {
   color: #461B7E;
+  max-width: 1200px;
+}
+
+.panel {
+  overflow: hidden;
+  max-width: auto;
+  text-align: center;
+}
+
+
+// Default medium-gray color
+$medium-gray: #cacaca;
+// Default global padding
+$global-padding: 1rem 3rem;
+// Global value used for all elements that have a border radius.
+$global-radius: 5px;
+
+// Setting the form border color
+$form-border-color: $medium-gray;
+// Setting the form padding
+$form-padding: $global-padding;
+// Setting the form border radius
+$form-radius: $global-radius;
+
+.log-in-form {
+  border: 1px solid $form-border-color;
+  padding: $form-padding;
+  border-radius: $form-radius;
+  background-color: white;
+}
+
+.page-title {
+  padding-left: 3rem;
 }
 
 /*** for About_Michelle ****/
+.michelle-background-image {
+  background: image-url('background_picture.jpg');
+  padding: 10px 5px 10px 5px;
+}
+
+.about-m {
+  max-width: 500px;
+}
 
 .item-wrapper {
   padding: 20px;
   border: 2px solid #efefef;
   border-radius: 7px;
-  margin-top: 40px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  background-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.